webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+
+module.exports = {
+  entry: __dirname + '/src/main.jsx',
+  output: {
+    path: __dirname + '/build/js',
+    filename: '[name].bundle.js'
+  },
+  externals: {
+    'react': 'React',
```

Comment:
`webpack -p` helps quite a bit but the build times are *really* slow:

Without externals `webpack`:
```
main.bundle.js – 929k @ 2214ms
```

With externals `webpack`:
```
main.bundle.js – 240k @ 1699ms
```

---

Without externals `webpack -p`:
```
main.bundle.js – 281k @ 9127ms
```

With externals `webpack -p`:
```
main.bundle.js – 98k @ 4843ms
```